sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobServiceImpl.java

```diff
@@ -2197,6 +2205,7 @@ private BlockBlobsStageBlockResponse stageBlockWithRestResponseIntern(String con
                                                                           byte[] blockContent,
                                                                           byte[] transactionalContentMD5,
                                                                           byte[] transactionalContentCrc64,
+                                                                          boolean computeMd5,
```

Comment:
I wonder if this should be a Boolean instead of boolean?

Reply:
True, I think Boolean is easier for people

Reply:
Fixed